src/global-messages/types.ts

```diff
@@ -21,6 +23,8 @@ export type GlobalMessageRaw = {
   appPlatforms: AppPlatformType[];
   appVersionMin: string;
   appVersionMax: string;
+  startDate?: Timestamp;
```

Comment:
For better encapsulation of Firestore, I think it is better to have `number` here. I know we expose the Firestore `Timestamp` type on some ticket types as well, but in the spirit of easter: "Det er aldri for sent å snu".